core/src/main/java/io/aiven/kafka/tieredstorage/chunkmanager/ChunkKey.java

```diff
@@ -16,20 +16,17 @@
 
 package io.aiven.kafka.tieredstorage.chunkmanager;
 
+import java.nio.file.Path;
 import java.util.Objects;
 
-import org.apache.kafka.common.Uuid;
-import org.apache.kafka.server.log.remote.storage.RemoteLogSegmentId;
-
 public class ChunkKey {
-    public final Uuid uuid;
+    public final Path segmentFileName;
```

Comment:
Should we change this to be just `String` instead? Since we have in memory cache also and from it's perspective path looks weird same as file name unless we say specifically that it's an s3 filename somewhere.